### ISSUE TITLE
Charter needs to allow for competitive solutions not exclusively controlled by browsers

### ISSUE THREAD

**Issue by jwrosewell** [patcg/proposals#9]
We heard yesterday how proposals are being limited by the stated position of browser vendors.

Suppose a better solution could be identified if the innovators in this group did not feel constrained?

The charter needs to unambiguously allow for solutions not exclusively controlled by browser vendors.

**Comment by martinthomson** [comment]
> We heard yesterday how proposals are being limited by the stated position of browser vendors.

No.  We heard opinions yesterday.

While I can't speak for what we will implement, Mozilla values the standards process and will respect its outcome.  That isn't a commitment to implement anything - we can't make that commitment; especially if the outcome is not in the interests of our users - this is a negotiation and I want it to be clear that we are entering into that negotiation in good faith.  My presentation yesterday was in the interests of being open about our starting position, that is all.

**Comment by alextcone** [comment]
The IPA proposal was written by Meta (not a browser) and Firefox (a browser who you want around to keep giving Chrome and Safari a run for their $.

**Comment by AramZS** [comment]
Our charter establishes our purpose:


> ... is to incubate web features and APIs that support advertising while acting in the interests of users, in particular providing strong privacy assurances.

> The group welcomes participation from browser vendors, advertisers, publishers, ad buyers, advertising platforms and intermediaries, privacy advocates, web application developers, and other interested parties. 

I am unclear how this might constrain us to stated positions of browser vendors. We've already had a presentation by a non-browser-vendor participant. 

I think the charter makes it clear we are open to all input and intend to work towards outputs considering any proposals that are technically relevant and proposal effective solutions.

**Comment by seanturner** [comment]
We should probably also transfer this one to: https://github.com/patcg/patcg.github.io/issues

**Comment by jwrosewell** [comment]
@martinthomson I was thinking of another talented engineer who confirmed they did not consider options that did not align to browser vendors stated positions. That is damaging for innovation.

@alextcone does IPA not rely on the browser to provide support and as a minimum not interfere with it?

@AramZS - question for browser vendors. Will they confirm they will not interfere with solutions that do not require them to make changes?

**Comment by alextcone** [comment]
> @alextcone does IPA not rely on the browser to provide support and as a minimum not interfere with it?

Yes. All of these proposals at the W3C include browsers in some way. IPA distributes the systems though to include non-browser entities. This is also true for PARAKEET, MaCAW and Masked LARk (and even FLEDGE to some degree).

**Comment by seanturner** [comment]
> @AramZS - question for browser vendors. Will they confirm they will not interfere with solutions that do not require them to make changes?

Can you clarify what you mean by "interfere"?
